src/Orbit/Models/Power System/Battery.cs

```diff
@@ -0,0 +1,12 @@
+﻿ using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orbit.Models.Power_System
+{
+    public class Battery
+    {
+        public double temperature;
+        public double chargeLevel;
```

Comment:
Can you just update these to PascalCase and make them properties rather than fields

e.g.
```c#
public double Temperature { get; set; }
```